homeassistant/components/linknlink/entity.py

```diff
@@ -0,0 +1,29 @@
+"""LinknLink Entities."""
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import LinknLinkCoordinator
+
+
+class LinknLinkEntity(CoordinatorEntity[LinknLinkCoordinator]):
+    """To manage the device info."""
+
+    _attr_has_entity_name = True
+
+    def __init__(self, coordinator: LinknLinkCoordinator) -> None:
+        """Initialize coordinator entity."""
+        super().__init__(coordinator)
+        self.api = coordinator.api
+        self.fw_version = coordinator.fw_version
+
+    @property
+    def device_info(self) -> DeviceInfo:
```

Comment:
This can be a shorthand attribute created in the constructor `_attr_device_info = DeviceInfo(..`